debian/enterprise-control-center/include/etc/confluent/docker/admin.properties.template

```diff
@@ -1,4 +1,14 @@
-{% set security_props = env_to_props('CONTROL_CENTER_STREAMS_PRODUCER_', '') -%}
+{% set producer_security_props = env_to_props('CONTROL_CENTER_STREAMS_PRODUCER_', '') -%}
+{% set security_props = env_to_props('CONTROL_CENTER_STREAMS_', '') -%}
 {% for name, value in security_props.iteritems() -%}
+{% if not name in producer_security_props -%}
 {{name}}={{value}}
+{% endif %}
+
+{# producer specific overrides -#}
+{% if len(producer_security_props) > 0 -%}
```

Comment:
why do we need this check?

Reply:
forgot to remove it, I had a comment in there that I didn't want to write out unless we had any overrides